waiverdb/templates/permissions.html

```diff
@@ -21,19 +21,23 @@
                     {% for p in permissions -%}
                     <tr>
                         <td>{{ p["name"] }}</td>
-                        <td>{{ p["maintainers"] | map("urlize") | join(" ") | safe }}</td>
-                        <td>{{ p["testcases"] | join("<br />") | replace('-', '&#8209;') | safe }}</td>
+                        <td><ul>
+                          {%- for maintainer in p["maintainers"] %}<li>{{ maintainer | urlize }}</li>{% endfor -%}
+                        </ul></td>
+                        <td><ul>
+                          {%- if p["_testcase_regex_pattern"] -%}<li><code>{{ p["_testcase_regex_pattern"] }}</code></li>{%- endif -%}
+                          {%- for testcase in p["testcases_ignore"] %}<li><b>NOT</b> <code>{{ testcase }}</code></li>{% endfor -%}
+                          {%- for testcase in p["testcases"] %}<li><code>{{ testcase }}</code></li>{% endfor -%}
+                        </ul></td>
                         <td>
                             {% if p["groups"] | length > 0 -%}
-                            <span class="font-weight-bold">Groups:</span><br />
-                            {{ p["groups"] | join("<br />") | replace('-', '&#8209;') | safe }}
-                                {% if p["users"] | length > 0 -%}
-                                <br />
-                                {%- endif %}
+                            <b>Groups:</b>
+                            <ul>{%- for group in p["groups"] %}<li>{{ group }}</li>{% endfor -%}</ul>
                             {%- endif %}
+
                             {% if p["users"] | length > 0 -%}
-                            <span class="font-weight-bold">Users:</span><br />
-                            {{ p["users"] | join("<br />") | replace('-', '&#8209;') | safe }}
+                            <b>Users:</b>
+                            <ul>{%- for group in p["users"] %}<li>{{ group }}</li>{% endfor -%}</ul>
```

Comment:
Just a small copy-paste mistake
```suggestion
                            <ul>{%- for user in p["users"] %}<li>{{ user }}</li>{% endfor -%}</ul>
```